pkg/queue/sharedmain/main.go

```diff
@@ -189,7 +189,8 @@ func Main(opts ...Option) error {
 	d.Transport = buildTransport(env)
 
 	if env.TracingConfigBackend != tracingconfig.None {
-		oct := tracing.NewOpenCensusTracer(tracing.WithExporterFull(env.ServingPod, env.ServingPodIP, logger))
+		tracingName := fmt.Sprintf("%s-%s-queue", env.ServingNamespace, env.ServingService)
```

Comment:
Not all revisions have a parent Knative Service.

Users can create configurations and then point routes to them.

Reply:
Good point, what do you think should be used in this case? Should we default to the default string if any of these two fields are not set?